libr/core/cmd.c

```diff
@@ -1188,6 +1188,22 @@ static int cmd_ls(void *data, const char *input) { // "ls"
 			eprintf ("Usage: less [file]\n");
 		}
 		break;
+	case 'j': { // "lj"
+		PJ * pj = pj_new();
```

Comment:
missing space before (

Reply:
> I see multiple problems with this PR:
> 
>     1. I believe that in json an object `{..}` needs keys and values. Right now you are just putting a value inside an object (`{"             file1                 ../                  ./               file2\n"}`) which, I think, is wrong.
> 
>     2. What is the benefit of having `lj` if it just returns the whole string returned by `l`? The right approach here would be to return a json array containing separate elements for each file. Something like: `["file1", "file2", ....]`.
> 
> 
> Does it make sense?

Yes, it does.
How should I do that? I tried some ways, but couldn't make it happen.
I'm not sure, but I think it should work if there's a comma in between each of the file names in the string? Currently, PJ thinks that this is a one big object.